src/components/BuildKiteBuildView/components/ActionOutput.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 RoadieHQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  LinearProgress,
+  Typography,
+} from '@material-ui/core';
+import { Progress } from '@backstage/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Alert from '@material-ui/lab/Alert';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import moment from 'moment';
+import React, { FC, Suspense } from 'react';
+import { useLog } from '../../useLog';
+
+const LazyLog = React.lazy(() => import('react-lazylog/build/LazyLog'));
+moment.relativeTimeThreshold('ss', 0);
+
+const useStyles = makeStyles({
+  accordion: {
+    marginTop: '0!important',
+  },
+  accordionDetails: {
+    padding: 0,
+  },
+  button: {
+    order: -1,
+    marginRight: 0,
+    marginLeft: '-20px',
+  },
+});
+
+export const ActionOutput: FC<{
+  url: string;
+  job: any;
+  className: string;
+}> = ({ url, job, className }) => {
+  const classes = useStyles();
+  const splitUrl = url.split('/');
+  const { loading, value, error } = useLog({ owner: splitUrl[5], repo: splitUrl[7], buildNumber: parseInt(splitUrl[9], 10), buildId: splitUrl[11] });
```

Comment:
As much as the named object as a param helps with readability why not just pass in the build url as a param? or have a util function which parses the url into an object.